spec/factories/posts.rb

```diff
@@ -1,5 +1,6 @@
 FactoryGirl.define do
   factory :post do |f|
     f.message "New post"
+    f.user_id 1
```

Comment:
how can you pass in a user rather than a user id? What would be the advantage of this? 
How can you get the post factory to  use the user factory?